pkg/services/system_intakes.go

```diff
@@ -32,6 +32,38 @@ func NewFetchSystemIntakesByEuaID(
 	}
 }
 
+// NewCreateSystemIntake is a service to create a business case
+func NewCreateSystemIntake(
+	config Config,
+	create func(intake *models.SystemIntake) (*models.SystemIntake, error),
+) func(context context.Context, intake *models.SystemIntake) (*models.SystemIntake, error) {
+	return func(context context.Context, intake *models.SystemIntake) (*models.SystemIntake, error) {
+		user, ok := appcontext.User(context)
+		if !ok {
+			// Default to failure to authorize and create a quick audit log
+			config.logger.With(zap.Bool("Authorized", false)).
+				With(zap.String("Operation", "CreateSystemIntake")).
+				Info("something went wrong fetching the eua id from the context")
+			return &models.SystemIntake{}, &apperrors.UnauthorizedError{}
+		}
+		intake.EUAUserID = user.EUAUserID
```

Comment:
We're setting this in the handler as well. We should pick one location or justify it being in both places.